lambda/src/main/resources/db/migration/V29__add_consignment_ref_existing_rows.sql

```diff
@@ -0,0 +1,7 @@
+-- Update consignment table to create consignment reference for existing Consignment rows
+UPDATE "Consignment"
+SET "ConsignmentReference"=CONCAT('TDR-2020-OLD-', "ConsignmentSequence");
```

Comment:
If someone reads this without the context we have, they might be surprised that this isn't in the same format as the consignment references added by the API.

Could you add a comment explaining that we're using the sequence number rather than the base-25 encoded value because it's too hard to do the base 25 encoding in a database migration, but that's fine because this only affects test data in intg and staging at the moment?

Reply:
Ah good point, will do! :+1: